docs/system-data-definitions-common-types.md

```diff
@@ -0,0 +1,22 @@
+# SystemDataDefinitionsCommonTypes
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-SystemData-V1-01
+- RPC-SystemData-V1-02
+
+## Description
+
+Validates that system data references utilize the common types.
+
+## How to fix the violation
+
+Ensure that references to system data use the common types definition for system data.
```

Comment:
Curious: why we didn't provide valid and invalid examples?

Reply:
We don't have a set of required sections for the docs as far as I know. I can add them here though.